### PR TITLE
Add workflow to update ci files

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

**Comment by daviddavis** [comment]
@dkliban thought this might be helpful. if so, I can work on getting it ready for review/merge.

File: templates/github/.github/workflows/scripts/update_ci.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+set -eu
+
+remote="update-ci-destination"
+
+git remote add $remote "https://$GITHUB_USER:$GITHUB_TOKEN@github.com/pulp/{{ plugin_name }}.git"
+git fetch -q $remote
+
+if [ -z $1 ]; then
+  branches=$(git for-each-ref --format='%(refname:short)' refs/remotes/$remote/)
```
Comment (daviddavis) [inline templates/github/.github/workflows/scripts/update_ci.sh.j2:11]: Need to scope the branches. Probably the default branch and x.x branches.

File: README.md [hunk 2]
```diff
@@ -195,6 +195,8 @@ The following settings are stored in `template_config.yml`.
                         The username and token are used to push the Changelog and version bump commits
                         created by the release workflow. The default is 'pulpbot'.
 
+  sync_ci               Enables a nightly workflow to update the CI files.
```
Comment (fao89) [inline README.md:198]: sweet!

**Review by fao89 — APPROVED** [review]

**Comment by fao89** [comment]
Does it have any blockers?
Can we apply to pulp_ansible for testing?

**Comment by daviddavis** [comment]
Sure, let's try it out.

**Pull request merged**